views/js/controller/creator/views/property.js

```diff
@@ -129,17 +129,51 @@ function($, ui, DataBinder, templates){
         * @private
         */
         function propValidation() {
-            var $togglers;
+            const $togglers = $('#authoringBack, #saver, #previewer');
+
             $view.on('validated.group', function(e, isValid){
+                let classErrorAll = $container.find('span.validate-error');
+                let $testSection = $('.tlb-button-on').parents('.section').attr('id');
+                let testSectionId = '#' + $testSection;
+                let $propsSectionError = $('#section-props-'+ $testSection).find('span.validate-error');
+                let $propsItemError = $('.itemref-props').find('span.validate-error');
+                let $deleteButtonInCurrentSection = $(testSectionId).find('[data-testid="remove-section"]');
+                let $arrowUpButtonInCurrentSection = $(testSectionId).find('[data-testid="move-up-section"]');
+                let $arrowDownButtonInCurrentSection = $(testSectionId).find('[data-testid="move-down-section"]');
+                let $rudblocksButtonInCurrentSection = $(testSectionId).find('[data-testid="manage-rubric-blocks"]');
+
                 if(e.namespace === 'group'){
-                    $togglers = $('#test-creator .property-toggler, #saver');
-                    if(isValid === true){
-                        $togglers.removeClass('disabled');
+                   if (isValid && $propsItemError.length === 0 && $propsSectionError.length === 0 ) {
+                       $(testSectionId).removeClass('section-error');
+                       $deleteButtonInCurrentSection.removeClass('disabled');
+                       $arrowUpButtonInCurrentSection.removeClass('disabled');
+                       $arrowDownButtonInCurrentSection.removeClass('disabled');
+                       $rudblocksButtonInCurrentSection.removeClass('disabled')
+                           .unbind('click');
```

Comment:
Why we need this workaround with `.unbind('click');` ?

Reply:
I had to add this part because I added a `prevent default` on the click on that button when disabling buttons (otherwise I was still clicking when disabled) and it needs to be "un-binded" in order for it to work again when it is enabled again... 
Let me know if you can think of any other way that you think is better